services/app-web/src/components/Tabs/Tabs.tsx

```diff
@@ -39,12 +39,17 @@ export const Tabs = ({
                             role="tab"
                             aria-selected={activeTab === tab.name}
                         >
-                            <Heading className={styles['easi-tabs__tab-btn']}>
+                            <Heading
+                                className={styles['easi-tabs__tab-btn']}
+                                aria-description={
+                                    activeTab === tab.name ? 'active' : ''
```

Comment:
A couple things here:
1. `aria-description` is [usually used for long description](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description). I don't think this is proper usage.
2. It feels like the bug here is really something with the `aria-selected` behavior. I would look into why that selected* aria isn't working properly instead - that's what is supposed to announce what tab is in use in semantic HTML. I'm confused if this was always broken or maybe I broke it when I hooked up tabs with the URL routing. 
-  Here's some documentation on how tabs should work (I helped write this and can answer any questions) - or see [WCAG ](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/)or MDN docs.

Reply:
in case its helpful - I just did a recording myself of the simple tabs from WCAG in assitive labs. This is using no aria describe or aria labels - the word "selected" comes from that aria-selected field being set. 

https://github.com/Enterprise-CMCS/managed-care-review/assets/10750442/40d13f87-0d3f-49f4-b4be-0a5134b95b89

Reply:
Thanks for the links, Hana! I'll look into this some more. `aria-selected` on the tab was definitely the first thing I tried but I haven't been able to get it to announce selected yet

Reply:
I was able to get `aria-selected` read out with changes to the HTML. However, when a tab is first activated, JAWS doesn't finish announcing the tab before the page URL changes. I looked into trying to trying `e.preventDefault` and passing in `replace: true` to `navigate()`. However the only thing that worked was adding a timeout after the active tab is selected but I really don't think we actually want to do that. 

https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/2d01842c-6794-49da-8a8b-c84889f1d57e

Reply:
Yes that makes sense. I don't think this is a bug since our tabs also are doing entire page loads (and acting more link nav links). To me, that behavior looks right to me for tabs hacked to behave as links. We are losing focus and its cutting off because of the reroute.